pkgs/top-level/all-packages.nix

```diff
@@ -13889,18 +13889,17 @@ with pkgs;
   };
 
   inherit (callPackages ../development/compilers/crystal {
-    llvmPackages = if stdenv.system == "aarch64-darwin" then llvmPackages_11 else llvmPackages_10;
+    llvmPackages = llvmPackages_13;
```

Comment:
Something I don't get from the x86-darwing error is that it refers to clang-11, but shouldn't it refer to clang-13?

Maybe we could try sticking with llvmPackages_11 here? I'm not sure for the underlying reason, but the llvm package in nixpkgs points to 11.

Or we are missing some path to use llvmPackages_13.clang

Reply:
:/ Closest discussion I could find: https://discourse.nixos.org/t/nixpkgs-aarch64-darwin-build-m1-uses-wrong-old-clang-how-to-fix/ (in our case x86_64 instead of aarch64)

the suggested change adds clang-unwrapped to PATH https://github.com/NixOS/nixpkgs/blob/6de161729c81dc98e844793cc9c8fda29d5ef62a/pkgs/build-support/libredirect/default.nix#L35-L49

Someone on darwin needs to give it a try otherwise I'll just revert to llvmPackages_11 for x86_64

The nix build definitely downloaded clang-13:
```
...
/nix/store/5b4s502i0ifbivjfb0pwxjlf7vrwgr2v-clang-13.0.1-lib
...
/nix/store/dx1q2d5zh3f45y8amkisy5lbq9b48zsx-clang-13.0.1
...
/nix/store/myqkicxg2zg8wmc84k048hn6fznilqm9-clang-wrapper-13.0.1
...
```
> https://logs.nix.ci/?attempt_id=65e680ba-2ce4-4aae-8375-2b4107302113&key=nixos%2Fnixpkgs.195606